openedx_learning/apps/authoring/backup_restore/toml.py

```diff
@@ -83,6 +85,22 @@ def _get_toml_publishable_entity_table(
     else:
         published_table.add(tomlkit.comment("unpublished: no published_version_num"))
     entity_table.add("published", published_table)
+
+    if hasattr(entity, "container"):
+        container_table = tomlkit.table()
+        container_map = {
+            "section": "section",  # name of the container class : name of the toml table
+            "subsection": "subsection",
+            "unit": "unit",
```

Comment:
I'm not clear what the purpose of this map is, if all the values map to themselves?